for some reason, `#include "types.h"` in the source code includes
/usr/include/webp/types.h instead of common/scripting/core/types.h

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 911e35d..2525d0c 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1221,6 +1221,7 @@ add_executable( zdoom WIN32 MACOSX_BUNDLE
 	common/thirdparty/math/tanh.c
 	common/thirdparty/math/fastsin.cpp
 )
+target_include_directories( zdoom BEFORE PRIVATE common/scripting/core )
 
 set_source_files_properties( ${FASTMATH_SOURCES} PROPERTIES COMPILE_FLAGS ${ZD_FASTMATH_FLAG} )
 set_source_files_properties( xlat/parse_xlat.cpp PROPERTIES OBJECT_DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/xlat_parser.c" )
